dom.bs

```diff
@@ -3822,87 +3822,55 @@ the empty string instead, and then do as described below, switching on <a>contex
  <a>nodes</a>.
 </dl>
 
+<p>The <dfn export>contiguous exclusive {{Text}} nodes</dfn> of a <a>node</a> <var>node</var> are
+<var>node</var>, <var>node</var>'s <a>previous sibling</a> <a>exclusive <code>Text</code> node</a>,
+if any, and its <a>contiguous exclusive <code>Text</code> nodes</a>, and <var>node</var>'s
+<a>next sibling</a> <a>exclusive <code>Text</code> node</a>, if any, and its
+<a>contiguous exclusive <code>Text</code> nodes</a>, avoiding any duplicates.
+
 <p>The <dfn method for=Node><code>normalize()</code></dfn> method, when invoked, must run these
-steps for each {{Text}} <a>node</a> <a>descendant</a> of <a>context object</a>:
+steps for each <a>exclusive <code>Text</code> node</a> <var>node</var> <a>descendant</a> of
+<a>context object</a>:
 
 <ol>
- <li>Let <var>node</var> be the {{Text}}
- <a>node</a>
- <a>descendant</a>.
+ <li>Let <var>length</var> be <var>node</var>'s <a for=Node>length</a>.
 
- <li>Let <var>length</var> be <var>node</var>'s
- {{CharacterData/length}} attribute value.
+ <li>If <var>length</var> is zero, then <a>remove</a> <var>node</var> and continue with the next
+ <a>exclusive <code>Text</code> node</a>, if any.
 
- <li>If <var>length</var> is zero,
- <a>remove</a> <var>node</var> and
- continue with the next {{Text}}
- <a>node</a>, if any.
+ <li>Let <var>data</var> be the concatenation of the <a>data</a> of <var>node</var>'s
+ <a>contiguous exclusive <code>Text</code> nodes</a> (excluding itself), in <a>tree order</a>.
```

Comment:
I meant you could have a concept equivalent to what "contiguous exclusive `Text` nodes (excluding itself)" does in this concept. Since the previous sibling can't be an exclusive `Text` node and the node itself is excluded, all that's left is "following exclusive `Text` nodes".